internal/core/metadata/v2/application/device.go

```diff
@@ -49,3 +51,34 @@ func AddDevice(d models.Device, ctx context.Context, dic *di.Container) (id stri
 
 	return addedDevice.Id, nil
 }
+
+// DeleteDeviceById deletes the device by Id
+func DeleteDeviceById(id string, dic *di.Container) errors.EdgeX {
+	if id == "" {
+		return errors.NewCommonEdgeX(errors.KindContractInvalid, "id is empty", nil)
+	} else {
+		_, err := uuid.Parse(id)
+		if err != nil {
+			return errors.NewCommonEdgeX(errors.KindInvalidId, "failed to parse ID as an UUID", err)
+		}
```

Comment:
These lines doesn't need to be in `else` clause . `else` can be removed .

Reply:
You're right, removed the else clause.